code/game/objects/structures/flora.dm

```diff
@@ -359,6 +362,27 @@
 		if(do_after(user,3 SECONDS,target=src))
 			to_chat(user,span_notice("You finish trimming [src]."))
 			change_visual()
+	// SKYRAT EDIT START - OOK CHEE
+	if(icon_state == "monkeyplant")
+		var/obj/item/food/grown/banana/banana = I
+		if(!istype(I))
```

Comment:
you sure it's intended to check the type of `I`, not `banana`?

Reply:
oops